nixops/backends/none.py

```diff
@@ -31,6 +35,7 @@ class NoneState(MachineState):
     def get_type(cls):
         return "none"
 
+    provison_ssh_key = nixops.util.attr_property("provisionSSHKey", None, bool)
```

Comment:
```suggestion
    provision_ssh_key: bool = nixops.util.attr_property("provisionSSHKey", None, bool)
```

(and the fall-out of this rename)